src/diff.js

```diff
@@ -1,3 +1,12 @@
+import isEmpty from 'lodash.isempty';
+
+// https://github.com/Kong/kong/blob/0.14.0/kong/plugins/acl/schema.lua
```

Comment:
Is there something specific to these config keys that makes it possible for them to be treated as strings or arrays? I'm not seeing anything in the schema to indicate that, so I'm wondering if this bug is actually present for any plugin that uses arrays as config values.

Reply:
Looking at your comment jogged something in my brain. The schema says whitelist is an array, makes no mention of being a string. I think treating whitelist as a string or one element array is wrong, because a whitelist can have multiple elements in the array. I need to look at this case a little more.